data/trie/trieStorageManager.go

```diff
@@ -293,6 +288,12 @@ func (tsm *trieStorageManager) snapshot(msh marshal.Marshalizer, hsh hashing.Has
 			continue
 		}
 		db := tsm.getSnapshotDb(snapshot.newDb)
+		if check.IfNil(db) {
+			isSnapshotsBufferEmpty, keys = tsm.isSnapshotsBufferEmpty()
+
+			log.Error("trie storage manager: getSnapshotDb error")
```

Comment:
why is this an error?

Reply:
Removed the logging. If db==nil, the error is somewhere else, not here (It is logged in the getSnapshotDb func).